### PR TITLE
Update some readmes.

### PR DESCRIPTION
**Pull request by Dirbaio**

### CODE REVIEW

**Comment by eldruin** [comment]
Could you rebase this and #474?

**Comment by Dirbaio** [comment]
@eldruin done

File: embedded-hal/README.md [hunk 1]
```diff
@@ -9,9 +9,81 @@
...
+
+## Platform agnostic drivers
+
+You can find platform agnostic drivers built on top of `embedded-hal` on crates.io by [searching
+for the *embedded-hal* keyword](https://crates.io/keywords/embedded-hal).
+
+If you are writing a platform agnostic driver yourself you are highly encouraged to [add the
+embedded-hal keyword](https://doc.rust-lang.org/cargo/reference/manifest.html#package-metadata)
+to your crate before publishing it!
+//
```
Comment (eldruin) [inline embedded-hal/README.md:79]:
```suggestion

```

**Review by eldruin — COMMENTED** [review]
Looks good to me, thank you!

**Review by eldruin — APPROVED** [review]

**Pull request merged**